packages/opentelemetry-exporter-prometheus/src/PrometheusExporter.ts

```diff
@@ -54,7 +54,7 @@ export class PrometheusExporter implements MetricExporter {
    * @param callback Callback to be called after a server was started
    */
   constructor(config: ExporterConfig = {}, callback?: () => void) {
-    this._logger = config.logger || new api.NoopLogger();
+    this._diagLogger = getDiagLoggerFromConfig(config);
```

Comment:
This will fallback to the api.diag rather than the previous NoopLogger, so while the default api.diag is a Noop Diag Logger, if it is set this would be a change in functionality.